setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='pyramid_workflow',
-    version='1.0.0',
+    version='1.0.1',
```

Comment:
Confirmando: essa biblioteca é realmente nossa - e não um fork?

Se for fork, deveríamos identificar a versão com `geru` (ou `openco`).

Reply:
Vou colocar como debito tecnico no readme. Não vou conseguir corrigir isso agora. :-(